tuf/repository_tool.py

```diff
@@ -220,7 +220,8 @@ def __init__(self, repository_directory, metadata_directory,
   def writeall(self, consistent_snapshot=False):
     """
     <Purpose>
-      Write all the JSON Metadata objects to their corresponding files.
+      Write all the JSON Metadata objects to their corresponding files for
+      roles which have changed.
```

Comment:
Did this slip in by accident? I'm okay with leaving it there. The rest of the PR is very clear and this is a legitimate clarification of `writeall`'s purpose (see #958 and #964).

Reply:
Yes, it did. Whoops.